test/unit/backwards_compatibility.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { JobStatus } from '../../src/models/JobStatus';
+import { JobType } from '../../src/models/JobType';
+
+describe('test items for backwards compatibility', () => {
```

Comment:
Adding this test file for general backwards compatibility testing. Tests in this class will follow the format
`{version change was introduced}+, {test description}`

Reply:
I was think more of a single unit test in an existing class, but this works as well.